File: #03/Categoria 04/q02 - max3/max3.js
max3 = (a,b,c) => {
    if ((a >= b) && (a >= c)) return a
    else if ((b >= a) && (b >= c)) return b
    else return c
}

console.log(max3(1,2,3));
console.log(max3(1,100,3));
console.log(max3(1,2,100));
console.log(max3(1,1,1));
console.log(max3(1,2,2));
console.log(max3(12334,2,-232322));

File: #03/Categoria 04/q02 - max3/max3.hs
max3 :: Ord a => a -> a -> a -> a
max3 a b c
  | (a >= b) && (a >= c) = a
  | (b >= a) && (b >= c) = b
  | otherwise = c

main :: IO ()
main = do
    a <- readLn :: IO Int
    b <- readLn :: IO Int
    c <- readLn :: IO Int
    print $ max3 a b c